sql/core/src/test/resources/sql-tests/inputs/group-by.sql

```diff
@@ -30,8 +30,15 @@ SELECT a + 2, COUNT(b) FROM testData GROUP BY a + 1;
 SELECT a + 1 + 1, COUNT(b) FROM testData GROUP BY a + 1;
 
 -- Aggregate with nulls.
+--
+-- In SPARK-21871, we added code to check the actual bytecode size of gen'd methods. If the size
+-- goes over `hugeMethodLimit`, Spark fails to compile the methods and the execution also fails
+-- in a test mode. So, we explicitly turn off whole-stage codegen here.
+-- This guard can be removed if this issue fixed.
+SET spark.sql.codegen.wholeStage=false;
```

Comment:
Please change the value of `spark.sql.codegen.hugeMethodLimit`

Reply:
ok